controller/contour/configmap.go

```diff
@@ -28,6 +28,13 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+	// contourCfgMapName is the name of Contour's ConfigMap resource.
+	// [TODO] danehans: Remove and use contour.Name when
+	// https://github.com/projectcontour/contour/issues/2122 is fixed.
+	contourCfgMapName = "contour"
```

Comment:
Doesn't this declaration conflict with the declaration in `deployment.go` (which is in the same package)?

Reply:
```
controller/contour/deployment.go:38:2: contourCfgMapName redeclared in this block
	previous declaration at controller/contour/configmap.go:35:22
```